pkg/operator/ceph/cluster/cluster.go

```diff
@@ -453,29 +453,30 @@ func (c *cluster) preMonStartupActions(cephVersion cephver.CephVersion) error {
 // Basically, it is executed between the monitors and the manager sequence
 func (c *cluster) postMonStartupActions() error {
 	// Create CSI Kubernetes Secrets
-	err := csi.CreateCSISecrets(c.context, c.ClusterInfo)
-	if err != nil {
+	if err := csi.CreateCSISecrets(c.context, c.ClusterInfo); err != nil {
 		return errors.Wrap(err, "failed to create csi kubernetes secrets")
 	}
 
 	// Create crash collector Kubernetes Secret
-	err = nodedaemon.CreateCrashCollectorSecret(c.context, c.ClusterInfo)
-	if err != nil {
+	if err := nodedaemon.CreateCrashCollectorSecret(c.context, c.ClusterInfo); err != nil {
 		return errors.Wrap(err, "failed to create crash collector kubernetes secret")
 	}
 
 	// Create exporter Kubernetes Secret
-	err = nodedaemon.CreateExporterSecret(c.context, c.ClusterInfo)
-	if err != nil {
+	if err := nodedaemon.CreateExporterSecret(c.context, c.ClusterInfo); err != nil {
 		return errors.Wrap(err, "failed to create exporter kubernetes secret")
 	}
 
 	if err := c.configureMsgr2(); err != nil {
 		return errors.Wrap(err, "failed to configure msgr2")
 	}
 
-	crushRoot := client.GetCrushRootFromSpec(c.Spec)
-	if crushRoot != "default" {
+	// Set config store options
+	if err := c.updateConfigStoreFromCRD(c.Spec.CephConfig); err != nil {
+		return errors.Wrap(err, "")
+	}
+
+	if crushRoot := client.GetCrushRootFromSpec(c.Spec); crushRoot != "default" {
```

Comment:
While your syntax is of course valid, I would suggest that if we're not just checking for `err != nil`, it's more readable in the original form:
```suggestion
	crushRoot := client.GetCrushRootFromSpec(c.Spec)
	if crushRoot != "default" {
```